src/assets/simp/build/simp.spec

```diff
@@ -10,7 +10,6 @@ Buildarch: noarch
 Requires: createrepo
 Requires: facter
 Requires: lsb
-Requires: simp-adapter
 Requires: httpd >= 2.2
```

Comment:
Doesn't simp-extras also need simp-adapter?  Seems like the version range should be moved here